docs/pages/includes/install-linux.mdx

```diff
@@ -117,105 +121,134 @@ the customer portal and select the URL for your package of choice.
 
 Next, use the appropriate commands for your environment to install your package.
 
-For example, use the following commands to install Teleport on a machine with an
-x86-64 chip via tarball:
-
-
-```code
-$ curl https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-$ shasum -a 256 teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-# Verify that the checksums match
-$ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-$ cd teleport
-$ sudo ./install
-```
-
-For FedRAMP/FIPS-compliant installations of Teleport Enterprise, package URLs
-will be slightly different:
-
-```code
-$ curl https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
-$ shasum -a 256 teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
-# Verify that the checksums match
-$ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
-$ cd teleport
-$ sudo ./install
-```
+<Tabs>
+  <TabItem label="tar.gz">
+
+    For example, use the following commands to install Teleport on a machine with an
+    x86-64 chip via tarball. Adjust as needed for your system architecture:
+
+    ```code
+    $ curl https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
+    $ shasum -a 256 teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
+    # Verify that the checksums match
+    $ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
```

Comment:
```suggestion
Adjust the path provided to `curl` for the download option that matches your system architecture:

    ```code
    $ curl https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-{YOUR-SYSTEM-Architecture}-bin.tar.gz.sha256
    # <checksum> <filename>
    $ curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-{YOUR-SYSTEM-Architecture}-bin.tar.gz
    $ shasum -a 256 teleport-ent-v(=teleport.version=)-linux-{YOUR-SYSTEM-Architecture}-bin.tar.gz
    # Verify that the checksums match
    $ tar -xzf teleport-ent-v(=teleport.version=)-linux-*-bin.tar.gz
```

Reply:
👍